### DISCUSSION TITLE
"language" parameter for openai stt

### DISCUSSION THREAD

**Discussion by Raigedas** [open-webui/open-webui#5431] [Ideas]
**Is your feature request related to a problem? Please describe.**
when i use openai as an stt engine and i try to speak in english it kind of works but shows the text in random exotic languages instead of english.

**Describe the solution you'd like**
a way to set a specific language for the openai stt engine. 
i see from openai api reference that in addition to the required field "model" it is possible to pass optional "language" field:
https://platform.openai.com/docs/api-reference/audio/createTranscription
in source code i found where stt config is being built:
https://github.com/open-webui/open-webui/blob/main/backend/open_webui/apps/audio/main.py
can we add an optional "language" parameter.

**Additional context**
https://github.com/open-webui/open-webui/blob/main/backend/open_webui/apps/audio/main.py
https://platform.openai.com/docs/api-reference/audio/createTranscription

**Comment by Raigedas** [comment]
or i may be wrong... i hardcoded a different "language" field on my machine and i saw no effect in language detection... what helped is when i also added field "prompt":"english language". so, we may need not just one addition input "language" but two - "language" and "prompt".

**Comment by PhilosophiMoonbeam** [comment]
I am having this same issue, most prominently with Grok's OpenAI-compatible API on Whisper V3 and V3 Turbo.

Their V3 Distiled model is limited to English but has a higher word error rate. Ability to force the language setting would be greatly appreciated.
